src/js/trove/cmdline-lib.js

```diff
@@ -5,8 +5,13 @@
   theModule: function(runtime, _, uri) {
     return runtime.makeModuleReturn(
       {
-        "command-line-arguments": runtime.makeFunction(function() {
-          return runtime.ffi.makeList(runtime.getParam("command-line-arguments").map(runtime.makeString));
+          "command-line-arguments": runtime.makeFunction(function() {
+              var args = runtime.getParam("command-line-arguments");
+              if (args.length == 0) {
+                  args = ["<browser>"];
```

Comment:
Yes!

I made this change because in `trove/cmdline.arr` there is the following code:
```
all-cmdline-params = CL.command-line-arguments()
file-name = all-cmdline-params.first
```

This assumes that the list returned by `command-line-arguments()` isn't empty (which seemed reasonable to me) so I changed the function to always return a non-empty list.